tests/crunchydb_create_connect.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
+Documentation       To Verify Database Provisioning From Developers view
+Metadata            Version    0.0.1
+
+Library             SeleniumLibrary
+Resource            ../resources/keywords/deploy_instance_dev.resource
+
+Suite Setup         Set Library Search Order    SeleniumLibrary
+Suite Teardown      Tear Down The Test Suite
+Test Setup          Given The Browser Is On Openshift Home Screen
+Test Teardown       Close Browser
+
+
+*** Test Cases ***
+Scenario: Verify error message for invalid credentials on CrunchyDB
+    [Tags]    smoke    RHOD-49-2
```

Comment:
Please update the JIRA ID which seems incorrect

Reply:
So as per the consolidated Smoke Tests Sheet, we have only one scenario for Verifying error message for Invalid Credentials, and hence the corresponding Stage Jira ID for that test case is RHOD-49
Ideally I should have added two more identical Test Cases for Crunchy and Cockroach on Stage Jira and updated tags accordingly, but I didn't get a chance, so as a temporary placeholder I just marked the tags as RHOD-49-1 for MongoDB and RHOD-49-2 for CrunchyDB for now
These will be updated with subsequent PRs

Reply:
Thanks!